tests/relay_integration/test_sdk.py

```diff
@@ -127,32 +127,32 @@ def test_bind_organization_context(default_organization):
     }
 
 
+def add_context(scope, organization, **kwargs):
+    scope.set_tag("organization.test", "1")
+
+
 @no_silo_test
-@override_settings(SENTRY_PROJECT=1)
+@override_settings(SENTRY_PROJECT=1, SENTRY_ORGANIZATION_CONTEXT_HELPER=add_context)
 @django_db_all
-def test_bind_organization_context_with_callback(settings, default_organization):
+def test_bind_organization_context_with_callback(default_organization):
     create_default_projects()
     configure_sdk()
 
-    def add_context(scope, organization, **kwargs):
-        scope.set_tag("organization.test", "1")
-
-    settings.SENTRY_ORGANIZATION_CONTEXT_HELPER = add_context
     bind_organization_context(default_organization)
 
     assert Hub.current.scope._tags["organization.test"] == "1"
 
 
+def broken_add_context(scope, organization, **kwargs):
+    1 / 0
+
+
 @no_silo_test
-@override_settings(SENTRY_PROJECT=1)
+@override_settings(SENTRY_PROJECT=1, SENTRY_ORGANIZATION_CONTEXT_HELPER=broken_add_context)
 @django_db_all
-def test_bind_organization_context_with_callback_error(settings, default_organization):
+def test_bind_organization_context_with_callback_error(default_organization):
     configure_sdk()
 
-    def add_context(scope, organization, **kwargs):
-        1 / 0
-
-    settings.SENTRY_ORGANIZATION_CONTEXT_HELPER = add_context
```

Comment:
Neat, so this would never get rolled back?

Reply:
Yep! Something about the ordering of the fixture cleanup in python_django and the original silo decorator results in our silo mode globally being set to monolith for the remainder of the test run 😬